doc/Doxyfile

```diff
@@ -95,47 +95,22 @@ WARN_LOGFILE           =
 #---------------------------------------------------------------------------
 INPUT                  = ../src
 INPUT_ENCODING         = UTF-8
-FILE_PATTERNS          = *.c \
-                         *.cc \
-                         *.cxx \
-                         *.cpp \
-                         *.c++ \
-                         *.d \
-                         *.java \
-                         *.ii \
-                         *.ixx \
-                         *.ipp \
-                         *.i++ \
-                         *.inl \
-                         *.h \
-                         *.hh \
-                         *.hxx \
-                         *.hpp \
-                         *.h++ \
-                         *.idl \
-                         *.odl \
-                         *.cs \
-                         *.php \
-                         *.php3 \
-                         *.inc \
-                         *.m \
-                         *.mm \
-                         *.dox \
-                         *.py \
-                         *.f90 \
-                         *.f \
-                         *.vhd \
-                         *.vhdl \
-                         *.cu
+FILE_PATTERNS          = *.cu       \
```

Comment:
`*.cfg` and `*.tpl` and and all other bash/python scripts are currently not added. one could do so, but they are not complete right now either and become independent projects in many cases, too.

(also: python uses its own param syntax)